src/AddIns/Uno.UI.Lottie/LottieVisualSourceBase.cs

```diff
@@ -129,9 +143,13 @@ public Size Measure(Size availableSize)
 		private readonly Size CompositionSize = default;
 #pragma warning restore CA1805 // Do not initialize unnecessarily
 
-		private Task InnerUpdate(CancellationToken ct)
+		private async Task InnerUpdate(CancellationToken ct)
 		{
-			throw new NotSupportedException("Lottie on this platform is not supported yet.");
+#if !NET461
+			throw new NotImplementedException();
+#else
+			await Task.Yield();
+#endif
 		}
```

Comment:
Something is looking wrong here.

This method is called here:

https://github.com/unoplatform/uno/blob/da01d6cab79c21f5403a22636453e4a41e7d7cd4/src/AddIns/Uno.UI.Lottie/LottieVisualSourceBase.cs#L149

But never awaited.

Reply:
Yes, this part of the code is not particularly good and will have to be reworked.